packages/react/src/useUploadThing.ts

```diff
@@ -1,4 +1,4 @@
-import { use, useRef, useState } from "react";
```

Comment:
this could be

```ts
import React, { useRef, useState } from "react";

...

const promiseRef = useRef<...>()

...

if (typeof React.use === "function") ...
```

if you want

Reply:
As dumb as it is, I do kind of like this better.

Reply:
Aight changed it up